testCourse/questions/internalExternalManualGrade/addingNumbersCodeUpload/question.html

```diff
@@ -0,0 +1,54 @@
+
+<h2>Internal, External, and Manual Grading Method Combinations</h2>
+
+<section>
+  <p>This question combines "Add two numbers", an "Internal" grading question, and "Fibonacci function, file upload", an "External" grading question.</p>
+
+  <p>This question has been configured with "Internal", "External", and "Manual" grading methods on the info.json 'gradingMethods' property. The "Internal" and "External" grading methods run configured as usual. 
+  The `server.py` files from the two questions have been merged into one `server.py` to accept both the "Internal" and "External" parameters submitted by the student when a question is answered.</p>
+
+  <p>The "Manual" grading method leverages the pre-existing "Internal" grading submission and results, and the "External" grading submission, displayed on the page. No HTML or Python boilerplate has been added to allow for the additional "Manual" grading method. 
+    
+  NOTE: It is, at the moment, not possible to display the "External" grading results on the student question submission view if the "Internal" grading method is also enabled. Therefore, as the view is re-used for the manual grading instructor view, it would also not be able to view the "External" grading results.</p>
```

Comment:
```suggestion
  <p>The "Manual" grading method leverages the pre-existing "Internal" grading submission and results, and the "External" grading submission, displayed on the page. No HTML or Python boilerplate has been added to allow for the additional "Manual" grading method.</p>
    
  <p>NOTE: It is, at the moment, not possible to display the "External" grading results on the student question submission view if the "Internal" grading method is also enabled. Therefore, as the view is re-used for the manual grading instructor view, it would also not be able to view the "External" grading results.</p>
```